### PR TITLE
Fix rectangle drawing in 2D mode

### LINKED ISSUE #6853: Fix export rectangle in 2d mode
When exporting a layer, the user drawn rectangle is correctly shown in 3d-mode but is missing in 2d-mode.

3d (correct behaviour):
image

Test link: https://map.geo-rapp.org/#share=s-kQUoK7rttFcRYrIGHQvj1qVY0Vh

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

Fixes #6853 

Currently rectangle user drawing like the one drawn when exporting a portion of the imagery layer is broken in 2D mode. This PR fixes it.

### Test me

- Open [this CI link](http://ci.terria.io/main/#configUrl=https://terria-catalogs-public.storage.googleapis.com/geo-rapp/map-config/prod.json&share=s-nZWndrSs6NFPi160eCRDBYBw3A5) for `main` branch, try out the `Export` option for the workbench item
- Notice that in 3D mode, a rectangle is rendered on the map to show the drawn region
- Notice that this doesn't happen in 2D mode.
- Now try the same with [this branch](http://ci.terria.io/fix-2d-rectangle-drawing/#configUrl=https://terria-catalogs-public.storage.googleapis.com/geo-rapp/map-config/prod.json&share=s-nZWndrSs6NFPi160eCRDBYBw3A5).

Before (no rectangle):
image

After (rectangle shown):
image

### Checklist

- [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
- [ ] I've updated relevant documentation in `doc/`.
- [x] I've updated CHANGES.md with what I changed.
- [x] I've provided instructions in the PR description on how to test this PR.

### CODE REVIEW

File: lib/ReactViewModels/MouseCoords.ts [hunk 1]
```diff
@@ -196,10 +196,9 @@ export default class MouseCoords {
     const coordinates = Cartographic.fromDegrees(
       latLng.lng,
       latLng.lat,
-      undefined,
+      0,
       scratchCartographic
     );
-    coordinates.height = <any>undefined;
```
Comment (na9da) [inline lib/ReactViewModels/MouseCoords.ts:202]: Setting `height` as `undefined` resulted in some computations producing `NaN` values which broke the rectangle drawing.

**Review by nf-s — CHANGES_REQUESTED** [review]
3D Terrain and 2D modes are working perfectly, but I am having an issue with 3D Smooth mode, when I try to draw a rectangle extent, I get the following error on the first click

image

This may be out-of-scope of this PR - if so, let me know and I'll make a ticket 🙂

**Comment by na9da** [comment]
Good catch @nf-s 

There was a similar issue in ellipsoidal terrain mode. I have fixed it now - should work once the CI instance is updated.

image

**Review by nf-s — APPROVED** [review]
Great, thanks!

**Comment by na9da** [comment]
Thanks @nf-s

**Pull request merged**